src/main/java/com/jlr/ttl/poc/loadmanager/controllers/LocationController.java

```diff
@@ -1,30 +1,56 @@
 package com.jlr.ttl.poc.loadmanager.controllers;
 
 import com.jlr.ttl.poc.loadmanager.models.Location;
-import com.jlr.ttl.poc.loadmanager.models.Vehicle;
 import com.jlr.ttl.poc.loadmanager.repositories.LocationRepo;
-import com.jlr.ttl.poc.loadmanager.repositories.VehicleRepo;
 import com.jlr.ttl.poc.loadmanager.services.LocationService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 @RestController
-@RequestMapping(path = "/api/load_management/locations")
+@RequestMapping(path = "/load_management/locations")
 public class LocationController {
     @Autowired
     private LocationService locationService;
     @Autowired
     private LocationRepo locationRepo;
 
-    /*@GetMapping("/all")
+    @GetMapping("/all")
     public ResponseEntity<List<Location>> getAllLocations() {
-        List<Location> response = locationService.getAllLoctionService();
+        List<Location> response = locationService.getAllLocationService();
         return new ResponseEntity<>(response, HttpStatus.OK);
-    }*/
+    }
+
+    @GetMapping("/find_loc/{loc_code}")
+    public ResponseEntity<?> findLocation(@PathVariable(value = "loc_code") String loc_code) {
+        try {
+            return ResponseEntity.ok(locationService.findLocationService(loc_code));
+        } catch (IllegalStateException e) {
+            return ResponseEntity.ok("Location not existed!");
+        }
+    }
+
+    @PostMapping("/add-loc")
+    public ResponseEntity<String> addNewLocation(@RequestBody Location newLoc) {
```

Comment:
Please remove the addNewLocation and delLocation methods. Let's focus on getting the get methods complete first. (You can leave them here, just comment them out)